src/Resources/contao/dca/tl_wem_portfolio_feed.php

```diff
@@ -121,6 +128,38 @@
             'sql' => 'int(10) unsigned NOT NULL default 0',
             'relation' => ['type' => 'hasOne', 'load' => 'lazy'],
         ],
+        'readFromRemote' => [
+            'exclude' => true,
+            'filter' => true,
+            'flag' => 1,
+            'inputType' => 'checkbox',
+            'eval' => ['doNotCopy' => true, 'submitOnChange' => true],
+            'sql' => "char(1) NOT NULL default ''",
+        ],
+        'readFromRemoteUrl' => [
+            'exclude' => true,
+            'inputType' => 'text',
+            'eval' => ['rgxp'=>'url', 'decodeEntities'=>true, 'maxlength'=>2048, 'dcaPicker'=>true, 'tl_class'=>'w50'],
```

Comment:
What does `'dcaPicker'=>true` do in that case ? Contao's doc is quite shy on details.

Reply:
Wrong copy/paste, fixed in 4c83f666ea75b8a2b2f54949d18540e9dd8d22e3